### ISSUE TITLE
KSP gives compile warning

### ISSUE THREAD

**Issue by dkastl** [pgRouting/pgrouting#294]
With #293 a warning message appears when compiling KSP:

```
[ 83%] Building C object src/ksp/src/CMakeFiles/routing_ksp.dir/ksp.c.o
/home/daniel/Repository/pgRouting/pgrouting/src/ksp/src/ksp.c:40:0: warning: "DEBUG" redefined [enabled by default]
 #define DEBUG 1
 ^
In file included from /home/daniel/Repository/pgRouting/pgrouting/src/ksp/src/ksp.c:31:0:
/home/daniel/Repository/pgRouting/pgrouting/src/ksp/src/ksp.h:29:0: note: this is the location of the previous definition
 #define DEBUG
 ^
/home/daniel/Repository/pgRouting/pgrouting/src/ksp/src/ksp.c: In function ‘kspDBG’:
/home/daniel/Repository/pgRouting/pgrouting/src/ksp/src/ksp.c:50:2: warning: format not a string literal and no format arguments [-Wformat-security]
  elog(NOTICE, msg);
  ^
```

It seems the updated KSP source contains its own logger.

**Comment by cvvergara** [comment]
Fixed with  commit 5cdd106ebe

**Comment by woodbri** [comment]
Fixed in develop.
